assets/launcher/lang/sklmessages_ru_RU.properties

```diff
@@ -135,57 +135,57 @@ manager.editor.option.name.placeholder=Безымянная сборка
 manager.editor.option.version.name=Версия
 manager.editor.option.version.vanilla=Vanilla
 # dont translate that line ^. It looks terrible compared to untraslated modloader's labels / не переводи строчку сверху она выглядит ужасно так как названия загрузчиков модов не возможно перевести.
-manager.editor.option.directory.name=Директория игры
-manager.editor.option.directory.placeholder=Использовать директорию по умолчанию
-manager.editor.option.directory.info.label=Некоротые моды могут требовать установку для нахождения в папке .minecraft, чтобы использовать её,
-manager.editor.option.directory.info.link=Нажмите сюда.
+manager.editor.option.directory.name=Папка игры
+manager.editor.option.directory.placeholder=Использовать папку по умолчанию
+manager.editor.option.directory.info.label=Некоротые моды могут требовать установку для нахождения в папке .minecraft. Чтобы использовать её,
+manager.editor.option.directory.info.link=нажмите здесь.
```

Comment:
138, 139 – "folder" instead of "directory"
140 – i thought it would make more sense if we split it into two sentences
141 – why big letter + better translation